src/pages/NotificationsPage/NotificationPage.tsx

```
@@ -0,0 +1,192 @@
+export function NotificationsPage() {
+  const [activeTab, setActiveTab] = useState("all");
+  const [filterUnread, setFilterUnread] = useState(false);
+
+  return (
+    <PageLayout title="Notifications">
+      <div className="mx-auto max-w-4xl">
+        {/* Header with Tabs and Filter */}
+        <div className="mb-6 rounded-lg border border-gray-200 bg-white shadow-sm">
+          <div className="flex items-center justify-between border-b border-gray-200 p-4">
+            <h1 className="text-2xl font-bold text-gray-900">Notifications</h1>
+            <button className="text-sm font-medium text-primary hover:underline">
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)